### PR TITLE
Fix legacy REST API

### PR DESCRIPTION
**Pull request by chengwenxi**
**Summary**

* Add label for `/wasm/code/{codeId}`
* Fix amino register

### CODE REVIEW

File: x/wasm/internal/types/codec.go [hunk 1]
```diff
@@ -8,20 +8,21 @@ import (
 	govtypes "github.com/cosmos/cosmos-sdk/x/gov/types"
 )
 
+// RegisterLegacyAminoCodec registers the account types and interface
```
Comment (alpe) [inline x/wasm/internal/types/codec.go:11]: Duplicate doc.

**Review by alpe — APPROVED** [review]
Thanks for the fixes 💐

**Review by alpe — CHANGES_REQUESTED** [review]
Thanks a for the fixes 💐 
Can you please remove the duplicate method doc? Otherwise 👍

File: x/wasm/internal/types/codec.go (hunk 1 above)
Comment (chengwenxi) [reply x/wasm/internal/types/codec.go:11]: oh...

**Review by alpe — APPROVED** [review]
Perfect. Thanks a lot 🌷

**Pull request merged**